chapters/theoretical_foundations/sections/non-eudlidean-spaces/subsections/practical_considerations.tex

```diff
@@ -0,0 +1,100 @@
+    \caption{Non-Euclidean translation causing a misalignment of objects}
+    \label{fig:misaligned-wheel}
+\end{figure}
+
+The second option isn't unfortunately free of distortions as well.
+For example, consider two identical squares of side length $0.5$.
+The first one with the bottom-left corner at the point $(0,0)$ and the second one with the corresponding corner at $(0.5, 0.5)$.
+After porting to spherical geometry using the \autoref{eq:elliptic-porting}, we get squares with side lengths (listed counter-clockwise starting at the bottom edge):
+\begin{equation*}
+    0.500, \,
+    0.479, \,
+    0.479, \,
+    0.500
+\end{equation*}
+for the first square and with side lengths
+\begin{equation*}
+    0.480, \,
+    0.425, \,
+    0.425, \,
+    0.480
+\end{equation*}
+for the second square.
+The side lengths of the square have been calculated as the lengths of geodesics\footnote{This is the "great-circle distance" equal to $2 \arcsin{(c / 2)}$, where $c$ is the chord length.} between the square's vertices.
+As we can see, the side lengths of the ported square are no longer equal to each other, and the distortion increases as the square is farther away from the origin.
+This effect can be seen in \autoref{fig:bending-car}.
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.6\textwidth]{chapters/theoretical_foundations/sections/non-eudlidean-spaces/resources/bending-car.png}
+    \caption{Distrotions caused by porting to spherical space}
+    \label{fig:bending-car}
+\end{figure}
+
+To minimize the distortions in spherical space we employed the following method.
+Due to the periodic nature of the porting given by \autoref{eq:elliptic-porting}, the scene has to be confined inside a 3-dimensional ball of radius $2\pi$.
+Since the distortions increase as an object is farther from the origin, we decided to split the scene into two physical regions -- balls of radius $\pi$.
+Points $p$ in the first ball (centered at the origin) are mapped to spherical geometry using the mapping
+\begin{equation} \label{eq:spherical-port-1}
+    \mathcal{P}_{E,1}(p) = (p / \lVert p\rVert \sin(\lVert p\rVert), \cos(\lVert p\rVert))
+\end{equation}
+and points $p$ in the second ball (which is centered at a point $c$) using the formula
+\begin{equation}\label{eq:spherical-port-2}
+    \mathcal{P}_{E,2}(p) = (p' / \lVert p' \rVert \sin(\lVert p' \rVert), -\cos(\lVert p'\rVert)),
+\end{equation}
+where $p' = p - c$.
+In the 2-dimensional case, the regions become disks with radii of length $\pi$, and the effect of using \autoref{eq:spherical-port-1} and \autoref{eq:spherical-port-2} can be visualized as "wrapping" the first disk on the upper half of a unit sphere, and "wrapping" the second one on the lower half of the sphere.
+
+Dealing with distortions in hyperbolic space requires a more drastic approach because the scene we wished to port was potentially infinite.
+The main goal was to keep the distortions as small as possible near the camera.
```

Comment:
I think that if we wanted to keep the distortions as small as possible we would just set curve to -0.0001 and would basically have a euclidean space. Maybe write that the goal was to keep the distortions the same regardless of the camera position because it was unplayable far from 0,0,0

Reply:
changed it, now it's hopefully clearer

Reply:
I still don't think thats what we wanted. I think we  just wanted to limit the distortions but not to keep them as small as possible